store/database/migrate/postgres/0010_update_table_instances_labels.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE instances ADD COLUMN instance_labels JSONB NOT NULL DEFAULT '{"retain":"false"}';
```

Comment:
yes

Reply:
I missed this comment earlier.
I spent significant time on this yesterday - 
I had set default to '{}'. Then the query would look something like this
```
SELECT * FROM instances 
WHERE
...
AND
        instance_labels->>'retain' = 'false' 
        OR NOT (instance_labels ? 'retain')
```
Problem was the squirrel library was treating ? in `instance_labels ? 'retain'` as a parameter and converting it to $1 and thus creating a wrong query.
Looks like a limitation in library.
So went with DEFAULT '{"retain":"false"}' to avoid the ? postgres query